hacks/README.md

```diff
@@ -1,64 +1,90 @@
-# The hacks/* Scripts
+# The hacks/\* Scripts
 
 These scripts simplify builds, deployments or development.
 
 ## update-assisted-ui-lib
-Can be executed right after creating a new `assisted-ui-lib` [release](https://github.com/openshift-assisted/assisted-ui-lib/releases) to update it's version in the
+
+Can be executed right after creating a new `assisted-ui-lib`
+[release](https://github.com/openshift-assisted/assisted-ui-lib/releases) to update it's version in
+the
+
 - assisted-ui application
 - uhc-portal (OCM)
 
 **Please see `The Release Process` section bellow before using it.**
 
 ## patch-react-scripts.sh
-Called from the yarn's `postinstall` phase to watch changes under `node_modules` within `yarn start`.
 
-With this change, the `openshift-assisted-ui-lib` dependency can be updated on-the-fly during development.
+Called from the yarn's `postinstall` phase to watch changes under `node_modules` within
+`yarn start`.
+
+With this change, the `openshift-assisted-ui-lib` dependency can be updated on-the-fly during
+development.
 
 ## ocm-deployed-version.sh
+
 Used by QE to check recent version deployed in the staging environment.
 
 ## create-test-vm.sh
-Used to create a virtual machine from an ISO (like a discovery iso), tested in our development environments (no guarantee about other envs).
+
+Used to create a virtual machine from an ISO (like a discovery iso), tested in our development
+environments (no guarantee about other envs).
 
 See comments inside the script for further details.
 
 # The Release Process
```

Comment:
I suggest we could move the README for "Release Process" as a separate document, and remove it from the "hacks" README. And add this as a link in the main README.

Reply:
I'll open a task for this

Reply:
https://issues.redhat.com/browse/MGMT-11074